arbitrary-users-patch/happy-path/build_happy_path_image.sh

```diff
@@ -28,7 +28,7 @@ if [ "$1" == "--push" ]; then
 fi
 
 # Build image for happy-path tests with precashed mvn dependencies
-docker build -t "${NAME_FORMAT}/happy-path:${TAG}" --no-cache --build-arg TAG=${TAG} "${SCRIPT_DIR}"/
+docker build -t "${NAME_FORMAT}/happy-path:${TAG}" --no-cache --build-arg TAG="${TAG}" "${SCRIPT_DIR}"/
```

Comment:
not related to this PR but why happy-path image is located in the devfile registry? is it used in some QA devfiles?

Reply:
QE reuses the arbitrary-user-id patch to build the happy path image, so that it has the same modifications as regular devfile images (username, home dir, etc.)